### ISSUE TITLE
Ensure that list types are used consistently

### ISSUE THREAD

**Issue by johnwunder** [STIXProject/schemas#262]
In some cases there's a wrapper element (`<Indicators><Indicator>`) while in other cases there isn't (`<Indicator><Alternative_ID>`).

One approach or the other should be taken and followed consistently.

**Comment by jordan2175** [comment]
Examples
IndicatorType - Alternative_ID should be an array of Alternative_IDs
IndicatorType - Valid_Time_Position should be an array of Valid_Time_Postions

**Comment by jordan2175** [comment]
Another one:
SightingType - Reference should be an array of References

**Comment by johnwunder** [comment]
Yep. Which approach would you prefer? Or do you not care as long as it's consistent?

**Comment by jordan2175** [comment]
If the element contains an array of elements, meaning that it is plural, we should use the plural version of the field name.  This will make it much easier to understand what is going on and what is expected.  Right now when you read the docs you do not know if it includes an array of objects or is just a single object.  You have to hunt and peck through the XSDs to try and figure it out.  It is really painful...

**Comment by johnwunder** [comment]
So you're suggesting:

```xml
<Indicator>
  <Alternative_IDs>
    <Alternative_ID>1</Alternative_ID>
    <Alternative_ID>2</Alternative_ID>
  </Alternative_IDs>
</Indicator>
```

Rather than:

```xml
<Indicator>
  <Alternative_ID>1</Alternative_ID>
  <Alternative_ID>2</Alternative_ID>
</Indicator>
```

Right?

**Comment by jordan2175** [comment]
Yes... There should be some visual clue for the developers of producing clients and consuming clients that there might be more than one, aka an array of them..  In something like JSON I am guessing it would look like:

```
{
    "stixPackage" : {
        "indicators" : [ {
            "alternativeIds" : [
                "CV-8989-foo-bar-0002",
                "ABC-1234-foo-bar-0002"
            ],
        } ]
     }
}
```
